drag_and_drop_v2/public/js/view.js

```diff
@@ -50,13 +51,21 @@
         if (item.is_placed) {
             style.left = item.x_percent + "%";
             style.top = item.y_percent + "%";
+            tabindex = -1;  // If an item has been placed it can no longer be interacted with,
```

Comment:
@bradenmacdonald just to be clear, a tabindex of -1 only makes an item focusable by javascript, so unless you had some other control that would move focus to that item, -1 would have no affect for the user.  tabindex of 0 will add a non-focusable element (like a div) into the default tab order.  If it cannot be interacted with, I say that it shouldn't have a tabindex at all.  Even if it has useful information, the user can still access the info using standard navigation (just not TAB navigation.)  Sound good?